### ISSUE TITLE
Inline LICENSE file from github.com/gonum/license/LICENSE

### ISSUE THREAD

**Issue by liggitt** [gonum/graph#200]
Since vendoring only pulls in files in the referenced dependency, it would be really helpful to include at least the LICENSE file from github.com/gonum/license in each of the gonum repositories.

As an example, the godep and license verification script in kubernetes is giving me fits on the gonum/graph dependency, since it doesn't specify a LICENSE itself, and I cannot godep the gonum/license repo, since it contains no buildable go files.

**Comment by kortschak** [comment]
In the not too distant future the gonum repos will be merged into gonum/gonum which will also have the license. The only PRs we have been waiting on for this have recently been merged so I expect this will move soon.

**Comment by sbinet** [comment]
indeed, but perhaps we could also add a simple `doc.go` file that hangs some godoc-friendly documentation so that `godep` is happy ?

**Comment by kortschak** [comment]
The gonum/gonum repo now has a LICENSE file and a functional graph suite (the API is not yet stable, but then that is the issue here isn't it).
